tests/test_sortProps.py

```diff
@@ -0,0 +1,37 @@
+
+from honeybee_schema.model import Model
+from pydantic.schema import schema
+import pytest
+
+
+def test_sort_required_params():
```

Comment:
Not sure what this test is going to test. If you want to test you should test the method form `_openapi.py` otherwise one would change the code there and this test will still pass.

Reply:
Please see new commits https://github.com/ladybug-tools-in2/honeybee-schema/pull/21/commits/b63d459a8f9029ba94cdfe6995603e012823387d#diff-ba7c3e9e2070c02344c6d8386ea500c5R30-R31